### DISCUSSION TITLE
ITV Movies

### DISCUSSION THREAD

**Discussion by HippoBloke** [Ascoware/get-iplayer-automator#404] [Q&A]
Hi all

I can't seem to download ITV movies anymore - search doesn't work and if I use the PID it says not in cache.

Am I missing something somewhere ?

Hippo

**Comment by skovatch** [comment]
I think what's happening is that movies don't have a separate 'series' page off of itv.com/hub/shows -- you go directly to the page for the movie. The code that scraps the JSON info for the programs isn't expecting that, so it's likely ignoring the show metadata.

Please file a bug -- this shouldn't be too hard to fix. 

Note that you can use "Use current webpage" to get the show info and download it, even though the status will read "not in cache".

**Reply by HippoBloke** [reply]
Thanks that works - never thought to click start as it was showing 'not in cache' when I used the webpage option.

I'll file a bug request.
